AmplifyPlugins/Auth/AWSCognitoAuthPluginTests/HubEventTests/AuthHubEventHandlerTests.swift

```diff
@@ -12,12 +12,12 @@ import XCTest
 class AuthHubEventHandlerTests: XCTestCase {
 
     var authHandler: AuthHubEventHandler!
-    override func setUp() {
-        try? Amplify.configure()
+    override func setUpWithError() throws {
```

Comment:
As I was adding the other test files, I noticed `Amplify.configure()` was failing silently for every test after the first one.